src/main/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEvent.java

```diff
@@ -212,35 +213,36 @@ private CaseData getCaseData(final CaseData caseData, final DssCaseData dssCaseD
         List<ListValue<DssMessage>> listValues = new ArrayList<>();
 
         if (isNotEmpty(dssCaseData.getOtherInfoDocuments())) {
-            //For showing additional information page data on tab
             for (ListValue<EdgeCaseDocument> documentListValue : dssCaseData.getOtherInfoDocuments()) {
                 Document doc = documentListValue.getValue().getDocumentLink();
+                String documentComment = documentListValue.getValue().getComment();
                 doc.setCategoryId(DocumentType.DSS_OTHER.getCategory());
                 CaseworkerCICDocument caseworkerCICDocument = CaseworkerCICDocument.builder()
                     .documentLink(doc)
                     .documentCategory(DocumentType.DSS_OTHER)
+                    .documentEmailContent(documentComment)
                     .build();
 
                 if (!docList.contains(caseworkerCICDocument)) {
                     docList.add(caseworkerCICDocument);
                 }
 
-                final User caseworkerUser = idamService.retrieveUser(request.getHeader(AUTHORIZATION));
-
-                final DssMessage message = DssMessage.builder()
-                    .message(dssCaseData.getAdditionalInformation())
-                    .dateReceived(LocalDate.now())
-                    .receivedFrom(caseworkerUser.getUserDetails().getFullName())
-                    .documentRelevance(dssCaseData.getDocumentRelevance())
-                    .build();
-
-                final ListValue<DssMessage> listValue = ListValue
-                    .<DssMessage>builder()
-                    .id(UUID.randomUUID().toString())
-                    .value(message)
-                    .build();
-
-                listValues.add(listValue);
+                if (isNotBlank(dssCaseData.getAdditionalInformation())) {
```

Comment:
We do not want this in the for loop. that would mean for every document uploaded we add the same additional doc info to the messages tab. Since there is only one additional info text box, please have this outside of the if and the for loop since it is separate - see CicDssUpdateCaseEvent as an example